nearcore/src/config.rs

```diff
@@ -825,6 +825,7 @@ pub fn init_configs(
     }
 
     let mut config = Config::default();
+    config.tracked_shards = vec![0];
```

Comment:
btw - seems that the config is actually downloaded/changed in the next lines, so maybe we should move this later.